[2019-10-02 22:00:22,848] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_project_dag.Load_demographic_staging_table 2019-10-03T02:20:28.947985+00:00 [queued]>
[2019-10-02 22:00:22,863] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_project_dag.Load_demographic_staging_table 2019-10-03T02:20:28.947985+00:00 [queued]>
[2019-10-02 22:00:22,863] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-10-02 22:00:22,863] {taskinstance.py:839} INFO - Starting attempt 2 of 4
[2019-10-02 22:00:22,864] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-10-02 22:00:22,894] {taskinstance.py:859} INFO - Executing <Task(CsvCopyS3ToRedshiftOperator): Load_demographic_staging_table> on 2019-10-03T02:20:28.947985+00:00
[2019-10-02 22:00:22,895] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'capstone_project_dag', 'Load_demographic_staging_table', '2019-10-03T02:20:28.947985+00:00', '--job_id', '250', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/project_dag.py', '--cfg_path', '/var/folders/dr/9yvjlnm11xv25k2gknf2n3z00000gn/T/tmp0lnkk1h3']
[2019-10-02 22:00:23,431] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table /usr/local/lib/python3.7/site-packages/airflow/configuration.py:627: DeprecationWarning: You have two airflow.cfg files: /Users/victorialemay/airflow/airflow.cfg and /Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read /Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/airflow.cfg, and you should remove the other file
[2019-10-02 22:00:23,432] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   category=DeprecationWarning,
[2019-10-02 22:00:23,538] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table [2019-10-02 22:00:23,538] {settings.py:213} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=61386
[2019-10-02 22:00:23,748] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table [2019-10-02 22:00:23,748] {__init__.py:51} INFO - Using executor LocalExecutor
[2019-10-02 22:00:24,213] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table [2019-10-02 22:00:24,212] {dagbag.py:90} INFO - Filling up the DagBag from /Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/dags/project_dag.py
[2019-10-02 22:00:24,257] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table [2019-10-02 22:00:24,257] {cli.py:516} INFO - Running <TaskInstance: capstone_project_dag.Load_demographic_staging_table 2019-10-03T02:20:28.947985+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2019-10-02 22:00:24,292] {csv_copy_s3_to_redshift.py:35} INFO - Using CSV data file us-cities-demographics.csv from S3 bucket project-capstone
[2019-10-02 22:00:24,293] {csv_copy_s3_to_redshift.py:36} INFO - Inserting into destination Redshift table demographic_staging
[2019-10-02 22:00:24,321] {csv_copy_s3_to_redshift.py:42} INFO - Copying data from S3 to Redshift
[2019-10-02 22:00:24,334] {logging_mixin.py:95} INFO - [[34m2019-10-02 22:00:24,334[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster-1.c4i0rfzboubh.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-10-02 22:00:25,080] {logging_mixin.py:95} INFO - [[34m2019-10-02 22:00:25,080[0m] {[34mdbapi_hook.py:[0m171} INFO[0m - 
        COPY demographic_staging 
        FROM 's3://project-capstone/us-cities-demographics.csv'
        ACCESS_KEY_ID 'AKIASZIXFGD4JFXXFSEI'
        SECRET_ACCESS_KEY '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'
        CSV DELIMITER ';' IGNOREHEADER 1
    [0m
[2019-10-02 22:00:25,354] {taskinstance.py:1051} ERROR - S3ServiceException:The S3 bucket addressed by the query is in a different region from this cluster.,Status 301,Error PermanentRedirect,Rid 0F739BFB27875C05,ExtRid +2zZttmRF4gFJYxpXOo9NjohjrFUzahFfSONo6ajqy/WhIE4aOHEArlm2s5rV/NXACx+La07bGI=,CanRetry 1
DETAIL:  
  -----------------------------------------------
  error:  S3ServiceException:The S3 bucket addressed by the query is in a different region from this cluster.,Status 301,Error PermanentRedirect,Rid 0F739BFB27875C05,ExtRid +2zZttmRF4gFJYxpXOo9NjohjrFUzahFfSONo6ajqy/WhIE4aOHEArlm2s5rV/NXACx+La07bGI=,CanRetry 1
  code:      8001
  context:   Listing bucket=project-capstone prefix=us-cities-demographics.csv
  query:     651
  location:  s3_utility.cpp:597
  process:   padbmaster [pid=23947]
  -----------------------------------------------

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/plugins/operators/csv_copy_s3_to_redshift.py", line 52, in execute
    redshift.run(formatted_sql)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
    cur.execute(s)
psycopg2.errors.InternalError_: S3ServiceException:The S3 bucket addressed by the query is in a different region from this cluster.,Status 301,Error PermanentRedirect,Rid 0F739BFB27875C05,ExtRid +2zZttmRF4gFJYxpXOo9NjohjrFUzahFfSONo6ajqy/WhIE4aOHEArlm2s5rV/NXACx+La07bGI=,CanRetry 1
DETAIL:  
  -----------------------------------------------
  error:  S3ServiceException:The S3 bucket addressed by the query is in a different region from this cluster.,Status 301,Error PermanentRedirect,Rid 0F739BFB27875C05,ExtRid +2zZttmRF4gFJYxpXOo9NjohjrFUzahFfSONo6ajqy/WhIE4aOHEArlm2s5rV/NXACx+La07bGI=,CanRetry 1
  code:      8001
  context:   Listing bucket=project-capstone prefix=us-cities-demographics.csv
  query:     651
  location:  s3_utility.cpp:597
  process:   padbmaster [pid=23947]
  -----------------------------------------------


[2019-10-02 22:00:25,358] {taskinstance.py:1074} INFO - Marking task as UP_FOR_RETRY
[2019-10-02 22:00:25,383] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table Traceback (most recent call last):
[2019-10-02 22:00:25,383] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/usr/local/bin/airflow", line 32, in <module>
[2019-10-02 22:00:25,383] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     args.func(args)
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     return f(*args, **kwargs)
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     _run(args, dag, ti)
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     pool=args.pool,
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     return func(*args, **kwargs)
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     result = task_copy.execute(context=context)
[2019-10-02 22:00:25,384] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/plugins/operators/csv_copy_s3_to_redshift.py", line 52, in execute
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     redshift.run(formatted_sql)
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table     cur.execute(s)
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table psycopg2.errors.InternalError_: S3ServiceException:The S3 bucket addressed by the query is in a different region from this cluster.,Status 301,Error PermanentRedirect,Rid 0F739BFB27875C05,ExtRid +2zZttmRF4gFJYxpXOo9NjohjrFUzahFfSONo6ajqy/WhIE4aOHEArlm2s5rV/NXACx+La07bGI=,CanRetry 1
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table DETAIL:  
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   -----------------------------------------------
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   error:  S3ServiceException:The S3 bucket addressed by the query is in a different region from this cluster.,Status 301,Error PermanentRedirect,Rid 0F739BFB27875C05,ExtRid +2zZttmRF4gFJYxpXOo9NjohjrFUzahFfSONo6ajqy/WhIE4aOHEArlm2s5rV/NXACx+La07bGI=,CanRetry 1
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   code:      8001
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   context:   Listing bucket=project-capstone prefix=us-cities-demographics.csv
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   query:     651
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   location:  s3_utility.cpp:597
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   process:   padbmaster [pid=23947]
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table   -----------------------------------------------
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table 
[2019-10-02 22:00:25,385] {base_task_runner.py:115} INFO - Job 250: Subtask Load_demographic_staging_table 
[2019-10-02 22:00:27,794] {logging_mixin.py:95} INFO - [[34m2019-10-02 22:00:27,794[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
